data/entities.yaml

```diff
@@ -1374,9 +1374,9 @@
   - |
     Also allows manipulating composite values consisting of a
     collection of named fields.  For example, `[x = 2, y = "hi"]`
-    is a value of type `[x : int, y : text]`.  Individual fields
+    is a value of type `[x : int, y : text]`{=type}.  Individual fields
     can be projected using dot notation.  For example,
-    `let r = [y="hi", x=2] in r.x` has the value 2.  The order
+    `let r = [y="hi", x=2] in r.x` has the value `2`.  The order
```

Comment:
Annotate snippet?
```suggestion
    `let r = [y="hi", x=2] in r.x`{=snippet} has the value `2`.  The order
```

Reply:
Not if we want it to be parsed, maybe I should have named it `{=invalid}` or `{=non-swarm}`.